tests/components/FilterSearch.stories.tsx

```diff
@@ -42,6 +42,7 @@ const meta: Meta<typeof FilterSearch> = {
   args: {
     label: 'Filter',
     onDropdownInputChange: undefined,
+    onDropdownInputFocus: undefined,
```

Comment:
needs to be renamed. also, do we need to set this to undefined if it's just a decorator function?

Reply:
We don't need it. But doesn't hurt to leave it here